### PR TITLE
Added social networks buttons

### PR DESCRIPTION
**Pull request by Litsyshyn**
https://razom.atlassian.net/browse/DO-855?atlOrigin=eyJpIjoiNDBmZTI3ODZhYWMxNGY0M2FiZTg5OTdkZmRiMmZjZTkiLCJwIjoiaiJ9

### CODE REVIEW

**Comment by Litsyshyn** [comment]
I am not sure about classes, or no.... i am pretty sure that they are wrong or not necessery.

File: index.html [hunk 1]
```diff
@@ -461,7 +461,20 @@ <h1 class="section__title h1">
                         <span lang="en">About us</span>
                     </a>
                 </ul>
+                <ul class="footer__item navigation navigation_align-center navigation_white">
+
+                    <a class="navigation__item" href="https://www.facebook.com/DataOceanGroup/" target="_blank">
+                        <img class="footer__item" src="static/img/facebook.svg" class="header__item @media">
```
Comment (roman-tiukh) [inline index.html:467]: `@media`?

File: index.html [hunk 2]
```diff
@@ -461,7 +461,20 @@ <h1 class="section__title h1">
                         <span lang="en">About us</span>
                     </a>
                 </ul>
+                <ul class="footer__item navigation navigation_align-center navigation_white">
```
Comment (roman-tiukh) [inline index.html:464]: what is class `navigation_align-center`, i cant find this class in css or js files.

`navigation_align-left` same

File: index.html [hunk 3]
```diff
@@ -461,7 +461,20 @@ <h1 class="section__title h1">
...
+
+                    <a class="navigation__item" href="https://www.facebook.com/DataOceanGroup/" target="_blank">
```
Comment (roman-tiukh) [inline index.html:466]: a child component of `<ul>` element must be `<li>`

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
design:
image
this PR:
image

File: index.html (hunk 2 above)
Comment (Litsyshyn) [reply index.html:464]: `navigation_align-center`  is a copy\paste from previous. There was `center`  and i thought i can move objects on page, but it's not )

File: index.html (hunk 1 above)
Comment (Litsyshyn) [reply index.html:467]: it is about positions on page, i thought it is about spaces between social networks icons(pics)

File: index.html (hunk 2 above)
Comment (roman-tiukh) [reply index.html:464]: this question also concerns @AlenaYanish

File: index.html (hunk 3 above)
Comment (roman-tiukh) [reply index.html:466]: @AlenaYanish

File: index.html (hunk 1 above)
Comment (roman-tiukh) [reply index.html:467]: http://htmlbook.ru/css/value/media

**Comment by Litsyshyn** [comment]
@roman-tiukh updated.

File: index.html [hunk 4]
```diff
@@ -479,7 +479,21 @@ <h1 class="section__title h1">
...
+                        <img class="footer__item" src="static/img/facebook.svg">
+                        </a>
+                        <a href="https://www.linkedin.com/company/data-ocean/" target="_blank">
+                        <img class="footer__item" src="static/img/linkedin.svg">
+                        </a>
+                        <a href="mailto:info@dataocean.us" target="_blank">
+                        <img class="footer__item" src="static/img/email.svg">
+                        </a>
+                    </li>
+                </ul>
```
Comment (roman-tiukh) [inline index.html:494]:
```suggestion
                <div class="footer__item">
                        <a href="https://www.facebook.com/DataOceanGroup/" target="_blank">
                            <img class="footer__item" src="static/img/facebook.svg">
                        </a>
                        <a href="https://www.linkedin.com/company/data-ocean/" target="_blank">
                            <img class="footer__item" src="static/img/linkedin.svg">
                        </a>
                        <a href="mailto:info@dataocean.us" target="_blank">
                            <img class="footer__item" src="static/img/email.svg">
                        </a>
                </div>
```

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (Litsyshyn) [reply index.html:494]: I'll try to do like this before, but it was'nt work or work incorrect. I dont remember how exactly, but its does'nt.

**Pull request merged**